addons/sale/models/sale_order.py

```diff
@@ -273,6 +273,7 @@ def _rec_names_search(self):
         string="Amount Before Discount",
         compute='_compute_amount_undiscounted', digits=0)
     country_code = fields.Char(related='company_id.account_fiscal_country_id.code', string="Country code")
+    company_price_include = fields.Selection(related='company_id.account_price_include')
```

Comment:
Make it readonly. Changing it from a SO is not a good idea.

Reply:
It is readonly by default